src/claim/Claim.js

```diff
@@ -1,19 +1,26 @@
 const _ = require('lodash');
 const sjcl = require('sjcl');
 const { UserCollectableAttribute } = require('@identity.com/uca');
-const definitions = require('./definitions');
 const { services } = require('../services');
+const definitions = require('./definitions');
+const { schemaLoader } = require('../schemas/jsonSchema');
 
-const validIdentifiers = _.map(definitions, d => d.identifier);
+const { validIdentifiers } = schemaLoader;
 
-const getDefinition = (identifier, version) => (
+const findDefinition = (identifier, version) => (
   version ? _.find(definitions, { identifier, version }) : _.find(definitions, { identifier })
```

Comment:
I'm probably missing something.... But where the definitions are defined now? I see they have been removed from line 4

Reply:
From the schemas I think

Reply:
As the schema's get loaded - it recreates the old definitions on the fly...